markhor_bringup/launch/markhor_observation.launch

```diff
@@ -0,0 +1,10 @@
+<launch>
```

Comment:
`markhor_observation`? Maybe `markhor_sensors`? I'm not convinced if `observation` is the right name.

Reply:
This file is tough to name:
`markhor_sensors ` was the old file name that we had, but would you guess that the robot arm is included in the `sensors` file? The file is meant for the readiness test and will include all the score multipliers.
`readiness_test` or `victim_board` would purely represent Robocup terms that are unintuitive from the outside.
That is why I aimed for a broader name to represent a moment where the robot doesn't move and analyses the situation.

Reply:
Right, well if we don't put the arm inside this launch file, I think `markhor_sensors` is more representative. But that's not a hill I'd die on. And when are we not using those _sensors_ that's not in the Robocup context?

Reply:
Right, well if we don't put the arm inside this launch file, I think `markhor_sensors` is more representative. But that's not a hill I'd die on. And when are we not using those _sensors_ that's not in the Robocup context?

Reply:
I wanted to include the arm and gripper since the launch file was for the readiness test and it requires both the sensors and the those.
For now, none, but I would rather not name things more broadly than context based when I can. Might be too confusing here however.

Reply:
It's alright, finding a good name takes time. And it is not a real problem anyway, although documentation might be justified by this uncertainty.